src/libcadet/CMakeLists.txt

```diff
@@ -101,6 +101,11 @@ set(LIBCADET_BINDINGMODEL_SOURCES
 	${CMAKE_SOURCE_DIR}/src/libcadet/model/binding/HICWaterOnHydrophobicSurfacesBinding.cpp
 	${CMAKE_SOURCE_DIR}/src/libcadet/model/binding/HICConstantWaterActivityBinding.cpp
 )
+set(LIBCADET_PHASETRANSITIONMODELINTERFACE_HPP_
```

Comment:
rename to "LIBCADET_PHASETRANSITIONMODEL_SOURCES" in accordance with binding models source name

Reply:
Since we don't have external functions jet, we dont need to add a preprocess step like for binding and reaction (line 220-241), aren't we?